### DISCUSSION TITLE
Sort Lists elements by number of repositories to reorganize them

### DISCUSSION THREAD

**Discussion by vinceAmstoutz** [community/community#35768] [Other Feature Feedback, Questions, & Ideas]
Currently,  it's possible to sort [Lists (beta feature)](https://docs.github.com/en/get-started/exploring-projects-on-github/saving-repositories-with-stars#organizing-starred-repositories-with-lists) by: 
<img width="227" alt="image" src="https://user-images.githubusercontent.com/46444652/195089139-5f6de7ef-3e19-4b0b-9cf5-3ca61b24f26c.png">

But in my personal case, it could be very relevant to sort by number of deposits by type (3 for Symfony and 1 for PHP so Symfony MAY appears before PHP), as you can see for example right here:

(This image is just an excerpt to show a quick sample and it is even more true when the list is a bit longer.).
<img width="437" alt="List of pinned repositories by type in GitHub profile" src="https://user-images.githubusercontent.com/46444652/195088658-e0108ea6-3a96-4e87-92ff-fa6e0cd6cab1.png">

It would be really useful when we consult the list because the more repositories we have for a type and the more potentially this type is used often. 

Or, otherwise, I think that we would need a feature like drag and drop to reorganize the items in the list as expected.

**Comment by AzizeBal** [comment]
> Currently,  it's possible to sort [Lists (beta feature)](https://docs.github.com/en/get-started/exploring-projects-on-github/saving-repositories-with-stars#organizing-starred-repositories-with-lists) by: 
> <img width="227" alt="image" src="https://user-images.githubusercontent.com/46444652/195089139-5f6de7ef-3e19-4b0b-9cf5-3ca61b24f26c.png">
> 
> But in my personal case, it could be very relevant to sort by number of deposits by type (3 for Symfony and 1 for PHP so Symfony MAY appears before PHP), as you can see for example right here:
> 
> (This image is just an excerpt to show a quick sample and it is even more true when the list is a bit longer.).
> <img width="437" alt="List of pinned repositories by type in GitHub profile" src="https://user-images.githubusercontent.com/46444652/195088658-e0108ea6-3a96-4e87-92ff-fa6e0cd6cab1.png">
> 
> It would be really useful when we consult the list because the more repositories we have for a type and the more potentially this type is used often. 
> 
> Or, otherwise, I think that we would need a feature like drag and drop to reorganize the items in the list as expected. 
> 
>

**Comment by jfullstackdev** [comment]
yup, Lists will be able to organize your content, just like the demo above

**Reply by vinceAmstoutz** [reply]
@jdevstatic What demo are you talking about please?

**Comment by vinceAmstoutz** [comment]
No answer and no way
